desktop/mac/release-notes/index.md

```diff
@@ -35,6 +35,43 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Security
 
+- CVE-2022-0847
+
+### New
+
+- Virtiofs released as an experimental feature
```

Comment:
Could we possible change this to:

* "Docker Desktop 4.6.0 gives macOS users the option of enabling a new experimental file sharing technology called virtiofs. During testing virtiofs has been shown to drastically reduce the time taken to sync changes between the host and VM, leading to substantial performance improvements for macOS users. As an example, enabling virtiofs has been shown to reduce the time taken to run DB imports by up to 98%."

"The Docker Desktop 'Volumes' UI now offers the ability to efficiently clean up volumes via multi-select checkboxes".

For volumes UI bulk removal, I unfortunately don't have enough context to flesh that out

Reply:
wonderful, this is perfect, thanks!